applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -570,13 +570,24 @@ void le_audio_adv_get(const struct bt_data **adv, size_t *adv_size, bool periodi
 	LOG_WRN("No advertiser in BIS headset");
 }
 
-int le_audio_play_pause(void)
+int le_audio_play(void)
+{
+	if (!paused) {
+		LOG_WRN("Already playing");
+		return -EAGAIN;
```

Comment:
Would `EALREADY` make more sense? `EAGAIN` implies that the request should be retried